lib/install.js

```diff
@@ -120,6 +120,11 @@ function handleResolvedPkg (dir, pkg, cb) {
       // package level.
       fs.rename.bind(null, tmpDest, finalDest)
     ], function (err) {
+      // Include package.json's metadata.
+      // `pkg` at this point just comes from the resolver, which only has
+      // { name, version, uid, shasum, tarball }.
+      pkg = assign({}, require(path.join(dir, pkg.uid, 'package.json')), pkg)
```

Comment:
I thought about another solution here.

We already have the `readPackage` function, here we can reuse its result.

Look at [this](https://github.com/alexanderGugel/ied/blob/master/lib/install.js#L48), here the `pkg` argument has proper information, then we need to pass it up to callback:

```js
function installSubDeps (tmpDest, dir, pkg, cb) {
  var subDependencies = assign({}, pkg.dependencies, pkg.peerDependencies)
  async.forEachOf(subDependencies, function (version, name, cb) {
    // ... cut ...
  }, function(err, result) {
      cb(err, result[0])
  })
}
```

Then we will have got package available [here](https://github.com/alexanderGugel/ied/blob/master/lib/install.js#L122) as the second argument.